### ISSUE TITLE
Current Test Suite

### ISSUE THREAD

**Issue by khronokernel** [OCLPWebDriver/OpenCore-Legacy-Patcher#1]
* [ ] Try kcditto
* [ ] Check KernelCache date modification on root volume as well as preboot

**Comment by flagersgit** [comment]
- Check Tesla drivers entry checked off
Observed behavior: Kext cache rebuilt properly and new kexts appeared properly.

**Comment by flagersgit** [comment]
- "Try sudo kcditto" entry checked off
	Observed behavior: as usual
- Check KernelCache date modification on root volume as well as preboot
	Observed behavior: as usual, date modified reflected cache rebuild time.

**Comment by flagersgit** [comment]
There are some clues as to how we should proceed in [this tonymacx86 article](https://www.tonymacx86.com/threads/discussion-about-how-to-make-nvidia-web-driver-works-in-macos-10-14-mojave.265813/).

It appears that they were close (?) to some sort of success but had issues with SkyLight.framework and MTL, which may be able to be solved by our heavily shimmed frameworks. They have detailed their patches, however they are a little hard to follow throughout the thread. Also, they have uploaded their binaries so we may be able to take a look at them.

Independent from their research, it appears that in 10.14.3 we are faced with this message spammed in the serial log:
`shim_io_connect_method_scalarI_structureO:4505 nvDevice: IOUserClient outputCount count mismatch 0x40 0x28 0xffffffff`

**Comment by flagersgit** [comment]
We have a new patch set in our binary repo that very closely tracks the TeraScale 2 patches that is in the mainline Dortania repo. Initial testing for this patch set will be done in Big Sur, and on my side specifically on version 11.2.3.
